### DISCUSSION TITLE
Support for UserInfo field?

### DISCUSSION THREAD

**Discussion by dbauducco** [alexandercerutti/passkit-generator#173] [Q&A] [answered]
Hey! Congratulations on creating/maintaining this project. Looks fantastic.

I took a look at the Wiki documentation to see if there was any support for setting the **userInfo** property. Can't seem to find it there. Is it supported but not documented, or just not developed yet? If it's not developed out, would y'all consider a PR request to add support in?

Here is a link to the Apple Documentation that covers it: https://developer.apple.com/documentation/walletpasses/pass

**Comment by dbauducco** [comment, marked as answer]
Obviously as soon as I click send, I choose to search through the code and find that it is here.
https://github.com/alexandercerutti/passkit-generator/blob/34a12e3879d38f7c4af83cb11433af049630efaa/src/schemas/index.ts#L143C5-L143C5

Seems like it's supported but would love confirmation! If you have any code examples, that would be greatly appreciated.

**Reply by alexandercerutti** [reply]
Hey @dbauducco thank you for using passkit-generator!

Of course, you found it!

I honestly never used it and I never understood how could I use it. So, just like nfc field, I added it in the attempt of support it.

Try to use it and, if you find any issue, feel free to open an issue and a PR to report the problem!


P.s. if you could leave a ⭐️ on the project, that would be awesome!
